DataTables/jyutping.cin

```diff
@@ -12677,4 +12678,718 @@ zyut6   拙
+jam1   䅧
+jam3   䅧
+jim3   䅧
+hong1  䆲
+deoi3  䇏
+mei4   䉠
+hung4  䉺
+jing1  䊔
+kiu4   䎗
+zyu1   䏭
+doek3  䐁
+duk1   䐁
+zaa1   䐁
+zaang1 㬹
+jaau4  䐓
+min5   腼
+tin2   腼
+ceon1  膥
+zaang1 䒏
+hong4  䒰
+jiu4   䔄
+deon1  䔻
+mei4   䕷
+mei4   䕷
+fai1   䘗
+gwan2  䘗
+pui3   䘗
+tou4   䟭
+sin6   䢭
+waan5  䤞
+kam4   䥅
+hin2   䥪
+hin2   䥪
+jik1   䦧
+wai6   䧥
+dik6   䨤
+taam4  䨵
+dang1  噔
+ming4  䫤
+jiu4   䬙
+bung2  䭰
+faai3  䭰
+zaai6  䭰
+wan4   䮝
+gang2  䱭
+kiu4   㝯
+ho2    㞹
+cuk1   㙇
+zaak6  讁
+na2    乸
+jing1  䣐
+man4   䪸
+long5  㮾
+jyu5   㦛
+jyu5   㝢
+tong1  劏
+gip3   喼
+zau3   㑳
+ging2  䜘
+gong1  㓻
+cing4  䝼
+jing6  䝼
+tiu4   䝼
+nip6   䯀
+seoi1  䯀
+wai2   䯀
+laa3   嚹
+long5  㙟
+ci4    䛐
+ngou6  㜜
+se1    㱔
+tam5   氹
+tam4   氹
+ji3    㦤
+maau5  㚹
+cyun2  䔿
+cyun3  䔿
+jyun2  䔿
+lok3   㦡
+lok6   㦡
+san1   㰑
+leoi4  㗊
+lo4    㗊
+gyun1  䣺
+dit6   䏲
+bit3   䏟
+gyun2  㷷
+tai3   䶑
+guk1   䶜
+syun3  䔉
+hot3   㓤
+kit3   㓤
+kit6   㓤
+zaan3  㜺
+kwaang3躀
+kwong3 躀
+hing3  㷫
+kaai2  㷫
+gau6   嚿
+me1    孭
+am2    揞
+fu5    㷆
+cong1  䱽
+coeng1 䱽
+je5    嘢
+cang1  罉
+saai1  嘥
+saai3  嘥
+aa1    吖
+ge3    嘅
+koi3   嘅
+naa4   嗱
+gaat6  曱
+aau1   㘭
+aau3   㘭
+zaat2  甴
+zaat6  甴
+go2    嗰
+go3    嗰
+hai2   喺
+zo2    咗
+di1    啲
+dit1   啲
+nung1  燶
+mang1  掹
+gwaa3  啩
+ham6   冚
+gam2   冚
+ngo4   㓟
+pei1   㓟
+lam1   冧
+nam1   冧
+dau2   唞
+tau2   唞
+zaang1 踭
+zang1  踭
+kam4   蠄
+jeon6  膶
+jeon2  膶
+mo4    䃺
+paak1  㕷
+gwaan1 㚬
+laap6  㧜
+lip6   㧜
+jyu1   㚥
+kam4   㩒
+syun4  䴉
+gwik1  䱛
+fong2  㨪
+gaang1 㹴
+kwai6  㙺
+cik1   䗩
+wong4  癀
+tik1   𧼮
+dap6   䂿
+zeoi1  䳡
+cim3   㟻
+ding1  㟻
+jam2   㟻
+tim2   㖭
+jyu4   䰻
+heoi1  㗾
+taap3  㗳
+kut3   䒷
+daam6  㗖
+kei4   䰇
+soi1   䰄
+ak1    戹
+laam2  㩜
+laam5  㩜
+oi2    㗒
+nik1   䁥
+bou3   㘵
+jat6   㳑
+to4    䭾
+zyu2   丶
+pit3   丿
+kyut3  亅
+tau4   亠
+gwing2 冂
+mik6   冖
+bing1  冫
+baau1  勹
+hai2   匸
+zit3   卩
+si1    厶
+seoi1  夊
+min4   宀
+jim2   广
+gai3   彐
+saam1  彡
+pok3   攴
+nik6   疒
+coek3  辵
+# leoi5	裏	# F9D8 inside, interior, within
+kau4   䊵
+din6   㙉
+tin2   㙉
+zeoi6  㑺
+zeon3  㑺
+hei1   㝃
+min5   㝃
+jau1   㳜
+caan2  剗
+lung4  㺪
+pok3   㺪
+sam3   㣌
+zeon3  㽙
+zoeng6 㽙
+jeoi6  㪫
+haai6  㪫
+hip6   叶
+ci2    㶴
+zik1   䳭
+zit1   䳭
+gaa1   䕒
+jyun4  㘣
+jau1   㚱
+hang1  㕡
+kok3   㕡
+je6    㖡
+git3   㛃
+lam5   㛦
+muk6   㜈
+hok6   㶅
+jing4  㜲
+mung4  䀄
+bou2   䴐
+cyu5   䣝
+tou4   䣝
+jin2   㠭
+faan4  㠶
+daai3  㯂
+jiu1   幺
+jan2   廴
+gwing2 㢠
+jau2   㫞
+tou1   㫞
+mei4   㵟
+wai6   㥣
+fui1   㷇
+ci4    㘹
+but6   㹀
+hap6   㤲
+hip3   㤲
+zit6   㨗
+zaa1   揸
+kwai4  㩦
+kwai4  㩗
+se2    䬷
+hang1  䃘
+mou4   无
+jin3   㬫
+si1    㴲
+ci5    㨁
+zi6    㨁
+zik6   㨁
+haam4  㮀
+ting1  䋼
+gun3   㮡
+king4  㯳
+lung4  㰍
+hou3   㝀
+taai5  舦
+daap6  㛥
+daap6  㳫
+taan2  㳫
+tong4  㜍
+maan6  䕕
+gaan2  㵎
+gaan3  㵎
+jin1   烟
+jan1   䄄
+cong4  牀
+fun1   㹕
+taai3  䣭
+nei6   㛅
+faai2  㻗
+pui5   㻗
+joek6  㜰
+kei4   䥓
+gaai3  鎅
+gei1   㼄
+hei3   㼄
+jin5   㼄
+kau5   㼄
+o1     疴
+aa2    瘂
+but6   癶
+gou1   㟸
+zan1   眞
+zoek6  着
+zoek3  着
+zaau1  着
+zaau2  着
+si1    㺨
+#
```

Comment:
I think you also have to move those comments out of the `%chardef` section or remove them entirely–for efficiency reasons OV parses the comments inside `%chardef` as actual definitions (so the comments become definitions of the key `#`).